src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/DeploymentOptions.cs

```diff
@@ -185,7 +191,7 @@ public class DeploymentOptions
                 ObjectType.LinkedServerLogins,
                 ObjectType.Endpoints,
                 ObjectType.ErrorMessages,
-                ObjectType.Filegroups,
+                ObjectType.Files,
                 ObjectType.Logins,
```

Comment:
Should we leave this to be explicit?

Reply:
Both will be present in options allowing user to change... only that Files will be excluded and groups included by default (following SSDT defaults).

(Files should not be by default included because they in most cases fail the deployment - one will have to explicitly exclude the changes regarding db files for it to work. Earlier it used to work because Filegroup meant both - files and filegroups)

Reply:
I see. Thanks makes sense. Thanks!